internal/github/github_test.go

```diff
@@ -0,0 +1,158 @@
+package github_test
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/openshift/backplane-cli/internal/github"
+	"github.com/openshift/backplane-cli/internal/upgrade"
+)
+
+func TestGetLatestVersion(t *testing.T) {
+
+	for name, tc := range map[string]struct {
+		Actual          upgrade.Release
+		ExpectedVersion string
+		ExpectedAssets  []upgrade.ReleaseAsset
+	}{
+		"no releases versions": {
+			Actual: upgrade.Release{
+				TagName: "",
+				Assets:  []upgrade.ReleaseAsset{},
+			},
+			ExpectedVersion: "",
+			ExpectedAssets:  []upgrade.ReleaseAsset{},
+		},
+		"latest releases versions": {
+			Actual: upgrade.Release{
+				TagName: "v0.0.1",
+				Assets:  []upgrade.ReleaseAsset{},
+			},
+			ExpectedVersion: "v0.0.1",
```

Comment:
The version here says expected 0.0.1 whereas at the info.go file it is 0.0.0, maybe a hardcoding thing again?